src/types.ts

```diff
@@ -1,9 +1,14 @@
 export type MeasureStartMessage = {
   msgType: 'measureStart';
-  width: number;
-  staves: number;
-  clefs: string[];
-  time: string;
+  width?: number;
+  staves?: number;
+};
+
+export type AttributesMessage = {
+  msgType: 'attributes';
+  clefs: Map<number, string>;
```

Comment:
Later on, I'd like to stick to types that are easily serializable to make it easy to send these messages over-the-wire (think multiple people editing the same MusicXML). No need to change now, but later on, I'd like this to be a `Record<number, string>`.

Reply:
OK